packages/flutter_tools/bin/podhelper.rb

```diff
@@ -99,12 +105,14 @@ def flutter_additional_ios_build_settings(target)
 def flutter_additional_macos_build_settings(target)
   return unless target.platform_name == :osx
 
-  # Return if it's not a Flutter plugin (transitive dependency).
-  return unless target.dependencies.any? { |dependency| dependency.name == 'FlutterMacOS' }
-
   # [target.deployment_target] is a [String] formatted as "10.8".
   deployment_target_major, deployment_target_minor = target.deployment_target.match(/(\d+).?(\d*)/).captures
```

Comment:
unrelated, but we should escape the `.` here, right? we can do it later if you want to hotfix this.

Reply:
Yes, human regex debugger.  I'll do that in master though as you say since I do want to hotfix this.